menpo/io/test/io_import_test.py

```diff
@@ -302,6 +302,24 @@ def test_importing_PIL_1_no_normalize(is_file, mock_image):
 
 @patch('PIL.Image.open')
 @patch('menpo.io.input.base.Path.is_file')
+def test_importing_PIL_1_proper_conversion(is_file, mock_image):
+    from menpo.image import BooleanImage
+ 
+    arr = np.zeros((10, 10), dtype=np.uint8)
+    arr[4, 4] = 255
```

Comment:
No problem, it's better to be thorough to avoid mistakes. :)

So, the reasoning with np.all is the following: arr is False in all positions apart from (4,4) (which is True). Then, with this change, I check that the new boolean image im has the same values in *all* positions as the arr. 
In other words np.all checks that im.pixels is everywhere False, apart from (4, 4). 

I don't think I have any mistake in the reasoning above but please let me know if i do.